### ISSUE TITLE
Error when mix JS files in production

### ISSUE THREAD

**Issue by pirmax** [JeffreyWay/laravel-mix#832]
- Laravel Mix Version: 0.12.0
- Node Version: 7.10.0
- NPM Version: 4.2.0
- OS: Windows 10

### Description:
I use Laravel Mix to mix CSS et JS files.

I use the command: **npm run production**

I have **undefined** in **combined.js** file.

But, the command **npm run dev** works.

I have this to mix:

```
mix.scripts([
    'public/backend/assets/js/jquery.min.js',
    'public/backend/assets/js/bootstrap.min.js',
    'public/backend/assets/js/detect.js',
    'public/backend/assets/js/fastclick.js',
    'public/backend/assets/js/jquery.slimscroll.js',
    'public/backend/assets/js/jquery.blockUI.js',
    'public/backend/assets/js/waves.js',
    'public/backend/assets/js/wow.min.js',
    'public/backend/assets/js/jquery.nicescroll.js',
    'public/backend/assets/js/jquery.scrollTo.min.js',
    'public/backend/plugins/switchery/switchery.min.js',
    'public/backend/plugins/bootstrap-sweetalert/sweet-alert.min.js',
    'public/backend/plugins/bootstrap-tagsinput/dist/bootstrap-tagsinput.min.js',
    'public/backend/plugins/waypoints/lib/jquery.waypoints.js',
    'public/backend/plugins/counterup/jquery.counterup.min.js',
    'public/backend/plugins/jquery-circliful/js/jquery.circliful.min.js',
    'public/backend/plugins/jquery-sparkline/jquery.sparkline.min.js',
    'public/backend/plugins/skyicons/skycons.min.js',
    'public/backend/assets/pages/jquery.dashboard.js',
    'public/backend/plugins/morris/morris.min.js',
    'public/backend/plugins/raphael/raphael-min.js',
    'public/backend/plugins/summernote/dist/summernote.min.js',
    'public/backend/assets/js/jquery.core.js',
    'public/backend/assets/js/jquery.app.js',
], 'public/backend/javascripts/combined.js');
```

### Steps To Reproduce:

To reproduce error, you must type this command: **npm run production**

**Comment by tdondich** [comment]
This is related to https://github.com/JeffreyWay/laravel-mix/issues/828 .  Note that specifying mix version 0.11.4 in your package.json resolves it until a fix is committed.

**Comment by pirmax** [comment]
Thanks, but now, I have this warning... **WARN: Output exceeds 32000 characters**

And I have error in Chrome console dev for JavaScript.

**Comment by tdondich** [comment]
That's a warning from uglify version being used.  But that should not cause any further issue in your JS.  I would suspect there might be an actual problem with your JS at this point?

**Comment by aios** [comment]
Problem still

**Comment by laracasts** [comment]
Fixed in v0.12.1.
